docs/utd_sysbio.config

```diff
@@ -0,0 +1,18 @@
+# nf-core/configs: UTD Sysbio Configuration
+
+All nf-core pipelines have been successfully configured for use on the Sysbio HPC cluster at the [The Univeristy of Texas at Dallas](https://www.utdallas.edu/).
+
+To use, run the pipeline with `-profile utd_sysbio`. This will download and launch the [`utd_sysbio.config`](../conf/utd_sysbio.config) which has been pre-configured with a setup suitable for the Sysbio HPC cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+
+Before running the pipeline you will need to load Singularity using the environment module system on Sysbio. You can do this by issuing the commands below:
+
+```bash
+## Singularity environment modules
+module purge
+module load singularity
+```
+
+All of the intermediate files required to run the pipeline will be stored in the `work/` directory. It is recommended to delete this directory after the pipeline has finished successfully because it can get quite large, and all of the main output files will be saved in the `results/` directory anyway.
```

Comment:
You could also consider adding `cleanup = true` to empty `work/` of files by default at the end of a successful run, if this could be an issue.